src/index.js

```diff
@@ -10,15 +10,19 @@ import './css/base.scss';
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
 //import './images/hotel.jpeg'
-
+let userID;
 //login functions
 $('#submit').click(event => {
   event.preventDefault();
 
   if ($('#username').val() === 'manager' && $('#password').val() === 'overlook2019') {
     window.location = './manager.html'
   }
-  let userID = parseInt($('#username').val().slice(-2));
+
+  if ($('#username').val().includes('customer')) {
+    userID = parseInt($('#username').val().split('r')[1]);
+    localStorage.setItem('userID', userID);
+  }
 
   if ($('#username').val().includes('customer') && $('#password').val() === 'overlook2019' && userID > 0 && userID <= 50) {
     window.location = './customer.html'
```

Comment:
At first I thought this was redundant code but I like how you used the ID to validate the login on line 27.